doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,122 @@
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Multitech Conduit AEP Gateway connected to the internet and running the [latest firmware](http://www.multitech.net/developer/downloads/).
+
+> This article addresses the minimum configuration required to connect your Multitech Conduit AEP model to {{% tts %}}.
+> For the first-time hardware and software setup, you can refer to the appropriate configuration guide on the [Multitech documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).
+
+> Note: The gateway EUI can be found at the bottom of the gateway under the field &quot;NODE LORA&quot;.
+
+## Configuration using a Terminal
+
+Firstly, you will need to generate a `global_conf.json` file required to add a custom channel plan to your gateway. Follow the steps below to generate the required json file.
+
+> **Warning:** The manual configuration of the frequency plan - required to make it work on the The Things (Enterprise) Stack - may result in a mismatch for some of the frequency bands used for the Multitech Gateway.
+
+The Gateway Configuration Server can be used to generate a proper `global_conf.json` configuration file for your gateway. You will need a Gateway API key with the `View gateway information` right enabled. The instructions for the same can be found in the relevant sections of the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}) getting started guides.
+
+Open the command prompt in Windows or any Linux terminal to run a curl command (as shown below) to generate the required `global_conf.json` file in your current working directory.
+
+Make sure you replace `thethings.example.com` with your server address:
+
+```bash
+$ curl -XGET \
+    "https://thethings.example.com/api/v3/gcs/gateways/{GATEWAY_ID}/semtechudp/global_conf.json" \
+    -H "Authorization: Bearer {GTW_API_KEY}" > ~/global_conf.json
+```
+
+> Note: Replace the required fields in the above command and run it.
+
+Once the `global_conf.json` file is generated, you will need to add this to your gateway. In a web browser, open the gateway’s configuration page by navigating to its IP Address obtained from the network it is connected to. Once logged in, you can configure the gateway to connect to {{%tts%}} by following the steps below:
+
+- Click on **LoRaWAN<sup>®</sup>** in the menu on the left. It opens the Gateway&apos;s configuration page.
+
+{{< figure src="005_Gateway_Menu_LoRaWAN.png" alt="MultiTech Conduit gateway home page" >}}
+
+- Under **Network Settings**, select the mode as &quot;Packet Forwarder&quot;.
```

Comment:
Please use bold instead of quotes for all of these and in the subsequent sections.

Reply:
@KrishnaIyer, 
Just to confirm for future reference, can **bold** be used for sections and subsection titles other than only for Buttons?
Ref: Comment by Mathieu Monneret (https://github.com/TheThingsNetwork/lorawan-stack/pull/1793#discussion_r368932472)